carlos-lopez/week7/challenge-cv/frontend/src/app/app.component.ts

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable no-empty-function */
+/* eslint-disable no-useless-constructor */
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable no-unused-vars */
+/* eslint-disable import/no-unresolved */
+import { Observable, Subject } from 'rxjs';
+import { OnInit, Component, AfterViewInit } from '@angular/core';
+import { switchMap, tap } from 'rxjs/operators';
+import { UserService } from './services/user.service';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.scss'],
+})
+// eslint-disable-next-line import/prefer-default-export
+export class AppComponent implements OnInit {
+  refresh$ = new Subject();
+
+  fetchCV$!: Observable<any>;
+
+  jobPosition = 'Full Stack Developer';
+
+  constructor(public userService: UserService) {}
+
+  ngOnInit() {
+    this.fetchCV$ = this.refresh$
+      .pipe(
+        switchMap(() => this.userService.fetchUsers()),
+      );
+  }
+
+  ngAfterViewInit() {
+    this.refresh$.next();
+  }
+
+  addExperience(
+    id:string, knowledgeArr:any, valueExp:string, valueYear: any, valueAcademy:any,
+  ) {
+    const myArr = knowledgeArr;
+    myArr.push({
+      language: valueExp,
```

Comment:
you could give the params the same name as the keys and you wouldn't have to double write them.